packages/agoric-cli/src/deploy.js

```diff
@@ -191,7 +191,7 @@ export default async function deployMain(progname, rawArgs, powers, opts) {
                 paths: [...paths, path.dirname(moduleFile)],
               });
             } catch (e) {
-              return path.resolve(...paths, fileName);
+              return path.resolve(path.dirname(moduleFile), ...paths, fileName);
```

Comment:
Why was `path.dirname(moduleFile)` added before `...paths` ?

Reply:
Nmv, this is correct and it's the `paths` for `require.resolve` above that isn't correct.